java/cli/build.gradle

```diff
@@ -16,11 +16,11 @@ repositories {
     jcenter()
     //mavenLocal()
     mavenCentral()
-    maven {
-        url 'http://artifacts.openmicroscopy.org/artifactory/ome.releases'
+    omeMaven {
```

Comment:
Pretty sure the `maven` here is telling Gradle what kind of repository this is not setting a name for it so you want that not `omeMaven`.